portfolio/src/main/webapp/index.html

```diff
@@ -59,6 +59,8 @@ <h1>Omar Montiel's Portfolio</h1>
       </div>
       <div id="contact" class="section">
         <a href="resume.pdf" target="_blank">Resume</a>
+        <button onclick="getMessage()">Try Fetch</button>
```

Comment:
Might be worth documenting a little here, it's hard for me to understand just by looking at your code what's happening here. At the very least make sure it's clear in your javascript what getMessage() is doing, but including some documentation here in the html wouldn't be the worst idea either!

Reply:
Sorry, will work into getting into that habit, for sure! I added the comments, can you let me know if they are OK, or if they need to be written differently?